src/chi/components/footer/footer.scss

```diff
@@ -0,0 +1,143 @@
+@import 'mixins';
+@import 'variables';
+
+.chi {
+  .o-footer {
+    align-items: center;
+    background-color: transparent;
+    box-shadow: 0 1px 2px 0 rgba(0, 0, 0, .04), 0 2px 2px 0 rgba(0, 0, 0, .12);
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    width: 100%;
+    z-index: 1;
+
+    &__content {
+      padding: 0 1.5rem;
+      position: relative;
+      width: 100%;
+    }
+
+    &__header {
+      color: set-color(black);
+      display: flex;
+      padding: 3rem 0;
+      position: relative;
+      width: 100%;
+      @include respond-to(sm) {
+        flex-direction: row;
+      }
+    }
+
+    &__footer {
+      align-items: center;
+      color: set-color(grey, 50);
+      display: flex;
+      font-size: $text-smaller;
+      font-weight: 600;
+      height: 4rem;
+      justify-content: space-between;
+
+      & .a-brand {
+        display: block;
+        width: 6rem;
+
+        & > img,
+        & > svg {
+          height: 1.5rem;
+          @include respond-to(sm) {
+            height: 2rem;
+          }
+        }
+        @include respond-to(sm) {
+          margin-right: 1rem;
+          width: 10rem;
+        }
+        @include respond-to(md) {
+          margin-right: 2rem;
+        }
+      }
+    }
+
+    &__start {
+      text-align: left;
+    }
+
+    &__end {
+      text-align: right;
+      text-decoration: none;
+      text-transform: capitalize;
+
+      .a-menu {
```

Comment:
@davidojedacl I think we can remove all this part from the footer code

Reply:
If i remove this part the menu links appears appear without styles and out of place
<img width="972" alt="screen shot 2019-01-04 at 11 01 24 am" src="https://user-images.githubusercontent.com/43439742/50682839-6e720c80-1010-11e9-9eae-2f96860fa779.png">